selfdrive/common/modeldata.h

```diff
@@ -1,10 +1,10 @@
 #pragma once
 
-#define MODEL_PATH_DISTANCE 192
-#define TRAJECTORY_SIZE 33
-#define MIN_DRAW_DISTANCE 10.0
-#define MAX_DRAW_DISTANCE 100.0
-#define POLYFIT_DEGREE 4
-#define SPEED_PERCENTILES 10
-#define DESIRE_PRED_SIZE 32
-#define OTHER_META_SIZE 4
+constexpr int MODEL_PATH_DISTANCE = 192;
```

Comment:
const should be the same here and it's shorter

Reply:
no special reason, just a quick copy and paste